### ISSUE TITLE
[BUG] wrong "info" tab displayed

### ISSUE THREAD

**Issue by vancleve** [ThomasFKJorna/zotero-night#88]
**Describe the bug**
When some PDFs are already open for reading, opening a new PDF in the viewer doesn't display correct "Info" tab. Rather, the "Info" tab displayed is for one of the already currently open PDFs.

**To Reproduce**
1. Open a PDF in the viewer
2. Return to the library and open another PDF
3. The "info" tab displayed is for the previous PDF and not for the PDF currently displayed in the viewer.

**Expected behavior**
PDF in the viewer and "info" tab correspond to the same Zotero item.

**Screenshots**
<img width="1672" alt="image" src="https://user-images.githubusercontent.com/1442334/206571913-d8b653d9-873a-410d-b520-95503fcb711a.png">

**OS:**
Mac OS 12.6

**Zotero Night version**
6.0.19-beta.6+3d8bbb3e5

**Comment by windingwind** [comment]
I guess this is a Zotero bug. From my own experience, if the PDF loading takes too long and multiple tab switches happen during this time, the right-sidebar may not be correctly displayed.

**Comment by vancleve** [comment]
I've never seen Zotero do this even with many PDFs open and lots of memory in use. I've only seen it when the plugin is activated.

**Comment by JunToronDoe** [comment]
I confirm that for me the bug appears when the plugin is enabled and disappear when it is disabled

**Comment by marcosmolla** [comment]
> I confirm that for me the bug appears when the plugin is enabled and disappear when it is disabled

Yes, I've just done the same test (with and without the plugin disabled and had the same result)
Zotero 6.0.21
Night for Zotero 0.4.20
on macOS 12.6.2

**Comment by lyh458** [comment]
Hi, there. I have the same issue.
OS: Ubuntu 20.04
zotero version: 6.0.20

**Comment by bmhj2** [comment]
Same issue here: incorrect information displayed when the plugin is active, not when it is disabled. I have been able
 to replicate the issue with all of the following systems:

- Ubuntu 21.10 and Zotero 6.0.20
- Windows 11 and Zotero 6.0.20
- Fedora 37 and Zotero 6.0.20

**Comment by MSBack** [comment]
I have the same issue on Windows. It happens every time I open a new tab, when other tabs are already open. I will disable Zotero night until this is addressed.

**Comment by maurice-schleussinger** [comment]
Can confirm on macOS 13.1 and Zotero 6.0.23.

I appreciate the work that has gone into this plugin and how it has saves my eyes in the later hours of the day. 🙂 

BUT, THIS BUG IS A MAJOR PROBLEM. I tend to extract cite-keys for publications from the item pane in the right and might end up citing the wrong item by mistake.

I suggest to add some kind of warning that is displayed during installation/use while this bug persists.

**Comment by marcosmolla** [comment]
Okay, so, I observed the following: when you have several tabs open, just switching between the last (in my case) five open tabs causes no problem. The info tab is correct. However, once the PDF viewer window needs to load a new PDF view (the window says 'Loading' for a second) the info pane will not be updated. 

I now see that this is what is already stated here: https://github.com/tefkah/zotero-night/issues/88#issuecomment-1377423561

**Comment by nathancashion** [comment]
> I confirm that for me the bug appears when the plugin is enabled and disappear when it is disabled

Same here. I disabled all plugins, couldn't recreate the issue. Enabled Night, and the problem reoccurred.

Pity, because I really like the dark mode. but I don't always remember to check before updating the item type or dates and have really messed up some references!
